code/datums/components/orbiter.dm

```diff
@@ -17,6 +17,12 @@
 
 	begin_orbit(orbiter, radius, clockwise, rotation_speed, rotation_segments, pre_rotation)
 
+/datum/component/orbiter/RegisterWithParent()
+	RegisterSignal(parent, COMSIG_MOVABLE_UPDATE_GLIDE_SIZE, .proc/orbiter_glide_size_update)
```

Comment:
orbiter already has register and unregister code, just 10 lines down

Reply:
*drools*